### DISCUSSION TITLE
change search by autoid

### DISCUSSION THREAD

**Discussion by harmkroog** [Budibase/budibase#10501] [Help]
Hi, ive upgraded my Budibase Version to Version 2.6.1
The auto search field is not anymore the display coloumn.
In screenshot.
My existing application it works.
![Auswahl_082](https://user-images.githubusercontent.com/130768693/236607977-45cf0426-c692-4e26-97a1-3d04008a0d66.png)

But ive cretaed an new and there not working anymore
It only searched the Auto ids.
![Auswahl_083](https://user-images.githubusercontent.com/130768693/236608060-585533cd-aa6c-4909-bc42-8e7e4627dea5.png)
If i left the table and go back then the display column is right.
But i would not serach by id.
I would search by kundenanzeige.
this is the display column in Table kunden
![Auswahl_085](https://user-images.githubusercontent.com/130768693/236608354-25ec9364-2cf2-40cb-979c-04b234de2de4.png)

what make iam wrong?
![Auswahl_084](https://user-images.githubusercontent.com/130768693/236608114-7f227ad1-8b76-47e4-8aa9-5d883105801e.png)

Edit in Modal, the Dropdown shows right. Its only in the Tabel view
![Auswahl_087](https://user-images.githubusercontent.com/130768693/236622618-ed6fb5b8-b660-46ab-8496-8726037d431b.png)

It seems is an bug in the table view.
But in open in Modal it works. So i edit it in modal until solution

Thank you!
Best regards Bonkersdleuxe

**Comment by ConorWebb96** [comment]
Hey @harmkroog,

Can you go to the table which that relationship is related to, and then ensure that the `Use as table display column` is toggled on for whatever column you wish to be displayed. (I've attached a screenshot below to help show this.)

Whatever column within a table has this toggled on, it will use that value as the display name for searches such as this.
<img width="381" alt="Screenshot 2023-05-09 at 11 16 20" src="https://github.com/Budibase/budibase/assets/126772285/5a34005e-b914-49f5-b139-7093193b852f">

Hopefully this makes sense and resolves your issue!

**Comment by harmkroog** [comment]
Hi thank you,
sorry i forgot to post that is this on.
In the full edit mode it will show right. The field kunden in the screenshot in the post before.
I think its an display eroor in table view.
The issue is not so relevant anymore because i could open the dataset in full view and there it works.
I dont see at the first the full screen edit mode.
Iam blind to see the little arrow button.
Ive got the new ui, after update, so i must finding me out again.
Thank you!
Best regards Bonkersdeluxe
